### ISSUE TITLE
Multi channel output support

### ISSUE THREAD

**Issue by Luca-DTU** [neuraloperator/neuraloperator#292]
It seems to me the current version of the code does not support multichannel output. The loss and the output processing normaliser assume single channel outputs, so the loss calls loss.item() and loss.backward(), which in multichannel leads to an error and should be substituted with loss.sum().item() or loss.mean.item(). Would also be nice at a later release to have support for adaptive loss weighting across the chanells via eg. softadapt.
In the normaliser the mean and std is calculated across the channels so if the magnitudes are different it screws up.

Multi channel output is relevant for plenty of applications in PDEs like predicting different velocity components in navier stokes. If there is a better way to approach this than multi-channel please let me know.
